package.json

```diff
@@ -51,4 +51,4 @@
   "dependencies": {
     "@bigtest/convergence": "^1.1.1"
   }
-}
+}
```

Comment:
Can you configure your editor not to strip the newline from the end of a file?

Reply:
Sorry about that. Updated my editor.